changelog/where.dd

```diff
@@ -0,0 +1,15 @@
+`~/dlang/install.sh` now supports where
+
+`where` has been added as a new option to the install script.
+It offers a convenient way to get the path of a specific compiler.
+Examples:
+
+$(CONSOLE curl https://dlang.org/install.sh | bash -s where
+/home/seb/dlang/dmd-2.078.2/linux/bin64
+)
+
+$(CONSOLE ~/dlang/install.sh ldc where
+/home/seb/dlang/ldc-1.7.0/bin
+)
+
+As with `install`, if a compiler isn't found, it will be installed.
```

Comment:
Why conflate these two concerns?  If one issues an instruction to `install`, then clearly one is asking for something to be installed if it isn't already.

If one is asking `where` something is then it seems reasonable that one might expect the response "nowhere", rather than have it be installed just so it can definitely be found.

Reply:
I think installation should be available as a flag for all commands which imply using a specific installer version.

I.e. `source ~/dlang/install.sh activate -i 2.080.0` should install 2.080.0 if it's not installed, then activate it.